codegenerator/cli/templates/static/codegen/src/LoadLayer.res

```diff
@@ -93,7 +101,7 @@ type rec t = {
     ~logger: Pino.t=?,
   ) => promise<array<Entities.internalEntity>>,
   makeLoadEntitiesByField: (
-    ~entityMod: module(Entities.InternalEntity),
+    . ~entityMod: module(Entities.InternalEntity),
```

Comment:
Is the `.` needed?

Reply:
No, but for some reason vscode extention started to formatting the file using the curried mode setting on. So to prevent breaking it, I've decided to add `.` here.